pydarn/plotting/power.py

```diff
@@ -0,0 +1,150 @@
+            default: 1100 kHz
+        statistical_calc: numpy object
+            numpy statistical calculation or generic min or max functions
+            e.g. numpy.std, numpy.median, numpy.min, numpy.max
+            default: numpy.mean
+
+        Raise
+        -----
+        NoDataFound
+        """
+
+        if compare:
+            # now compare frequencies separated by a frequency cutoff
+            # predefine the lists of separated records
+            low_freq_records = [record for record in records
+                                if record['tfreq'] < frequency]
+            high_freq_records = [record for record in records
+                                 if record['tfreq'] > frequency]
+
+            if len(low_freq_records) == 0 and high_freq_records == 0:
+                raise(exceptions.plot_exceptions.\
+                      NoDataFoundError('lag0', beam_num,
+                                       opt_beam_num=records[0]['bmnum']))
+            # gather important info regarding the records of interest
+            # get the date information from the first record
+            date = low_freq_records[0]['origin.time']
+            # get the site location
+            stid = high_freq_records[0]['stid']
+            # site location in terms of abbreviation
+            radar_abbrev = SuperDARNRadars.radars[stid].hardware_info.abbrev
+            # now compute the statistical statistical_calc of lag 0 power to be
+            # plotted in the high frequency records
+            for record in high_freq_records:
+                stat_pwr = statistical_calc(record['pwr0'])
+                record.update({'pwr0': stat_pwr})
+            for record in low_freq_records:
+                stat_pwr = statistical_calc(record['pwr0'])
+                record.update({'pwr0': stat_pwr})
+
+            plt.subplot(2, 1, 1)
+            RTP.plot_time_series(high_freq_records, parameter='pwr0',
+                                 beam_num=beam_num)
+            plt.title(' Lag 0 Power for {} Beam: {} '.format(radar_abbrev,
+                                                             beam_num))
+            plt.ylabel("{} Power\n [raw units]".format(statistical_calc))
+            plt.legend(["{} kHz".format(low_freq_records[0]['tfreq'])])
+            plt.xticks([])
+            plt.subplot(2, 1, 2)
+            RTP.plot_time_series(low_freq_records, parameter='pwr0',
+                                 beam_num=beam_num)
+            plt.ylabel("{} Power\n [raw units]".format(statistical_calc))
+            plt.legend(["{} kHz".format(low_freq_records[0]['tfreq'])])
+        else:
+            # get records of interest that have a specific frequency
+            records_of_interest = [record for record in records
+                                   if record['tfreq'] == frequency]
+
+            if len(records_of_interest) == 0:
+                raise(exceptions.plot_exceptions.\
+                      NoDataFoundError('lag0', beam_num,
```

Comment:
After testing, I found that this can error also when the frequency is incorrect, but the exception doesn't mention this, so I think the exception should be made more clear for this case, maybe like this?
```suggestion
                      NoDataFoundError('pwr0 for frequency {}'.format(frequency), beam_num,
```